### PR TITLE
Wire up I18n Formatter Checker to verify correct bundle keys & formatting for user-facing messages

### PR DESCRIPTION
**Pull request by PawelLipski**
We can consider incorporating this one directly into #407.

### CODE REVIEW

**Comment by PawelLipski** [comment]
> Unrel: It seems like GH has problem with proper commits range determination when base branch is force-pushed

@micpiotrowski yeah they would in fact need to do (at least) sth like `git merge-base --fork-point`, but it looks like they're just doing `git merge-base`

File: scripts/run-pre-build-checks [hunk 1]
```diff
@@ -13,7 +13,7 @@ bash ${x_option:+-x} ./scripts/enforce-change-notes-updated-on-pr-to-master
 ./scripts/enforce-newline-at-eof
 ./scripts/enforce-settings-gradle-match-directories
 ./scripts/enforce-to-string-call-super-in-subclasses
-./scripts/enforce-unique-key-shortcuts-for-action-texts
+############## ./scripts/enforce-unique-key-shortcuts-for-action-texts
```
Comment (PawelLipski) [inline scripts/run-pre-build-checks:16]: @mkondratek since #407 most shortcuts are repeated, but maybe it's okay... TBD what should be done with that

Comment (PawelLipski) [reply scripts/run-pre-build-checks:19]: (and this script was not failing since the path to bundle was no longer correct)

Comment (mkondratek) [reply scripts/run-pre-build-checks:20]: yeah, I think we can live without this script

File: frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardParentToMatchBranchAction.java [hunk 2]
```diff
@@ -77,34 +77,29 @@ public void onUpdate(AnActionEvent anActionEvent) {
...
 
       var parentName = gitMacheteNonRoot.getUpstreamBranch().getName();
-      presentation.setDescription(GitMacheteBundle
-          .message("action.GitMachete.BaseFastForwardParentToMatchBranchAction.description", parentName, branchName.get()));
+      presentation.setDescription(
+          format(getString("action.GitMachete.BaseFastForwardParentToMatchBranchAction.description"),
+              parentName, branchName.get()));
 
     } else {
+      // @formatter:off
```
Comment (mkondratek) [inline frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseFastForwardParentToMatchBranchAction.java:89]: :open_mouth:

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties [hunk 3]
```diff
@@ -3,8 +3,8 @@
 action.GitMachete.description.disabled.undefined.branch-name={0} disabled due to undefined branch name
 action.GitMachete.description.disabled.undefined.machete-branch={0} disabled due to undefined machete branch
 
-GitMachete.FetchBackgroundable.notification.fail=Fetch of refspec {0} failed
-GitMachete.FetchBackgroundable.notification.success=Fetch of refspec {0} succeeded
+action.GitMachete.FetchBackgroundable.notification.fail=Fetch of refspec {0} failed
```
Comment (mkondratek) [inline frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:6]: `FetchBackgroundable` is not an action :confused:

Comment (PawelLipski) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:6]: Hmmm it's true... but TBH shouldn't it then be `GitMachete.action.xxx` or `GitMachete.FetchBackgroundable.xxx`? Actually, do we even need that `GitMachete` prefix for anything? :thinking:

File: scripts/run-pre-build-checks (hunk 1 above)
Comment (PawelLipski) [reply scripts/run-pre-build-checks:20]: Removed

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties (hunk 3 above)
Comment (mkondratek) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:6]: We need `GitMachete` for bundles that are referenced by `plugin.xml` only

Comment (PawelLipski) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:6]: _We need_ = _without that prefix, we could get a name conflict with a property of the same key defined somewhere else in the classpath_, right?

Comment (PawelLipski) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:6]: Or is there any other reason?

Comment (PawelLipski) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:6]: Also, replacing `action.GitMachete.xxx` with `GitMachete.action.xxx` everywhere should still be okay wrt. plugin.xml, right?

**Pull request merged**